src/DictVectors/DictVectors.jl

```diff
@@ -5,46 +5,17 @@ dictionaries. The idea is to do linear algebra with data structures that are
 neither subtyped to `AbstractVector` nor to `AbstractDict` and are suitable
 for use with `KrylovKit.jl`. For this, the
 abstract type and interface [`AbstractDVec`](@ref) is provided, with the
-following concrete subtypes:
-
- * [`DVec`](@ref)
- * [`DFVec`](@ref)
- * [`FastDVec`](@ref)
+concrete implementation of [`DVec`](@ref)
 """
 module DictVectors
 
 using Random, LinearAlgebra
-#using DataStructures
-
-import LinearAlgebra: mul!, dot
-
-import Base: length, iterate, getindex, setindex, setindex!, get, get!, haskey,
-             getkey, pop!, isempty, empty, empty!, delete!, sizehint!,
-             zero, similar, eltype, ==, isequal, copy, copyto!, *, fill!,
-             values, keys, pairs, *
-# I don't know why we have to import == and isequal from Base. We are not
-# defining any new methods, but since removing the subtyping to AbstractDict,
-# suddenly we get errors like
-# MethodError: no method matching isequal(::Int64, ::Int64)
-# You may have intended to import Base.isequal
-
-
-# include("FastBufs.jl")
-using ..FastBufs
-
-export AbstractDVec, capacity, zero!, kvpairs, pairtype, add!, copytight
-export AbstractProjector, NormProjector, Norm2Projector, UniformProjector
-export Norm1ProjectorPPop
-export DVec, DVec2
-export DFVec, tuples, gettuple, flagtype, flags
-export FastDVec
+export AbstractDVec, zero!, add!, deposit!, storage
+export DVec, InitiatorDVec
+export AbstractProjector, NormProjector, Norm2Projector, UniformProjector, Norm1ProjectorPPop
 
 export StochasticStyle,
-    IsStochastic, IsDeterministic, IsStochasticWithThreshold,
-    IsDynamicSemistochastic,
-    @setThreshold, @setDeterministic, setThreshold
-    # IsStochasticNonlinear, IsStochastic2Pop,
-    # IsStochastic2PopInitiator, IsStochastic2PopWithThreshold, IsSemistochastic
+    IsStochastic, IsDeterministic, IsStochasticWithThreshold, IsDynamicSemistochastic, StyleUnknown
```

Comment:
@joachimbrand @mtsch is it a good idea to rename `IsStochastic` to `IsStochasticWithInteger`?

Reply:
Done.